### ISSUE TITLE
TCP Tracker Report Question 

### ISSUE THREAD

**Issue by SurbhiBakshi** [cityofaustin/atd-knack-data-tracker#261]
Still have some issues with inconsistent data in the TCP tracker. Also, it needs to be cleaned up for this years Average Review Time Report, because we only have 4 reviewers left in the ROW Division. So how it’s calculating the average is wrong.

Remaining Reviewers:
•	Shawn
•	Jon
•	Juliet
•	Doug

Month to date, does not match year to date for the same month?

**Comment by SurbhiBakshi** [comment]
Remaining Reviewers:
•	Shawn
•	Jon
•	Juliet
•	Doug

**Comment by SurbhiBakshi** [comment]
Month to Date in Average Review Time (Business Days) does not match Average Review Time by Month (Business Days) 
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/f0f9388b-89b0-4dcc-b7ff-514913deac62)

![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/3ae94217-ef40-43d8-a085-16960165363e)

**Comment by SurbhiBakshi** [comment]
Average Review Time (Business Days) 

Month to Date - Review Completed Date
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/cbf49e7c-38bd-4916-848b-6d7ba285e392)

**Comment by SurbhiBakshi** [comment]
Average Review Time by Month (Business Days) 

This Year - Submitted Date

![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/4d0ac377-5856-48a1-a9f9-8d38f1ff8f93)

**Comment by SurbhiBakshi** [comment]
Draft Report - Average Review Time (Business Days) 
Month to Date - Review_Completed_Date
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/89f570b7-0475-4549-a52a-c3065e0db828)

**Comment by SurbhiBakshi** [comment]
Draft Report - Average Review Time by Month (Business Days) 
This Year - Review_Completed_Date
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/933fc554-e11f-4644-8949-cda43afa30c4)

**Comment by SurbhiBakshi** [comment]
Another discrepancy I noticed while looking at this issue. Going to fix this. Not sure if this gets pulled in any report. 

![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/bd4f6375-c394-4a97-b856-b3598ac36c00)

**Comment by SurbhiBakshi** [comment]
A case has not been assigned a reviewer, although the reviw is complete - this is the existing discrepancy between the draft reports 

![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/500a6dbb-7c86-43ca-82eb-d78c1086059c)

**Comment by SurbhiBakshi** [comment]
@johnclary this was the same issue as last time, I am not sure why the report switched back to Submitted Date. Changing the filter to Review Completed Date has synced the reports. There is one unassigned case that accounts for the difference between the two numbers. 
I also changed the report to not show any empty records, and that took reduced the number of reviewers. Both reports are labelled Draft and are at the very end of the page.
Checked both reports over a few days to make sure the error doesn't get reintroduced.


If you want to take a look, and let me know if you see any more changes, I can then get back to Ryan. I will have to put in a note about the average calculations (since that is dependent on the number of reviewers and looks off especially at the start of the month).
